### ISSUE TITLE
Output many strange labels when training on VOC dataset?

### ISSUE THREAD

**Issue by rxqy** [pjreddie/darknet#884]
Hi, I'm now training on VOC dataset following the instructions on the homepage
I tried to download the pretrained weights, and I got the correct result with the following code
`./darknet detect cfg/yolov3.cfg weights/yolov3.weights data/dog.jpg`

However, when I'm training with yolo v3 on VOCdataset, the loss is indeed going down
`./darknet detector train cfg/voc.data cfg/yolov3-voc.cfg darknet53.conv.74`
And testd the weights after 300 iterations, I would see a lot of weird output labels,
like bees... skateboard? or sth like that
`./darknet detect cfg/yolov3.cfg backup/yolov3-voc_300.weights data/dog.jpg`

And I realized that these names should be comming from yolo9000.....
Is this the case, or am I having errors when training?

**Comment by rxqy** [comment]
Solved. Actually the labels are coming from coco.
The most straight forward way would be to set line 440 in darknet/example/darknet.c from
`test_detector("cfg/coco.data", argv[2], argv[3], filename, thresh, .5, outfile, fullscreen);` to
`test_detector("cfg/voc.data", argv[2], argv[3], filename, thresh, .5, outfile, fullscreen);`
